rega-genotype/src/main/java/rega/genotype/viruses/generic/GenericTool.java

```diff
@@ -207,6 +207,9 @@ private boolean subgenogroupPhyloAnalysis(AbstractSequence s, Result blastResult
 		PhyloClusterAnalysis.Result r = a.run(s);
 		ScanAnalysis.Result scanResult = checkBootScan(a, s);
 		
+		//System.out.println("r:"+r+"s:"+s+"-blastResult:"+blastResult+"-region:"+region+"-cutToRegion:"+cutToRegion+"-scanResult:"+scanResult);
+		
```

Comment:
Please remove all the changes to the whitespaces that have no effect on this file